app/generic/definitions.py

```diff
@@ -248,11 +257,69 @@ async def create_credential_definition(
             raise CloudApiException(
                 "Unable to construct credential definition id from signature response"
             ) from e
-
     else:
-        await stop_listener()
         credential_definition_id = result.credential_definition_id
 
+    if credential_definition.support_revocation:
```

Comment:
Does this need to be done manually? I know they autmoated 99 percent of it, and thought this was also handled, but not 100% sure

Reply:
Hmm let me double check this, but I didn't get this to work otherwise

Reply:
so not doing this leaves credential issuance in 'abandoned' state. 

I'll get to the bottom of it